pkg/dataplane/deployment.go

```diff
@@ -215,33 +214,34 @@ func (d *Deployer) ConditionalDeploy(
 				err.Error()))
 		}
 
-		if ansibleEE.Status.Succeeded > 0 {
+		if ansibleJob.Status.Succeeded > 0 {
 			log.Info(fmt.Sprintf("Condition %s ready", readyCondition))
 			nsConditions.Set(condition.TrueCondition(
 				readyCondition,
 				readyMessage))
-		} else if ansibleEE.Status.Active > 0 {
-			log.Info(fmt.Sprintf("AnsibleEE job is not yet completed: Execution: %s, Active pods: %d", ansibleEE.Name, ansibleEE.Status.Active))
+		} else if ansibleJob.Status.Active > 0 {
+			log.Info(fmt.Sprintf("AnsibleEE job is not yet completed: Execution: %s, Active pods: %d", ansibleJob.Name, ansibleJob.Status.Active))
```

Comment:
should this block also apply when `ansibleEE.Status.Failed <= *ansibleEE.Spec.BackoffLimit`? since "AnsibleEE job is not yet complete" is true in that case

Reply:
I think we should keep updating the status when the job fails, so the human operator can check if there is something wrong or just wait for the retry